release-notes/2.0/2.0.0-preview2-commit.md

```diff
@@ -138,7 +138,7 @@
 * [`[c02df73]`](https://github.com/dotnet/cli/commit/c02df73) Updating the websdk version for 1.0
 * [`[88b2e06]`](https://github.com/dotnet/cli/commit/88b2e06) Merged host and SDK help
 * [`[b44bce5]`](https://github.com/dotnet/cli/commit/b44bce5) Fix dotnet store options
-* [`[357fd3d]`](https://github.com/dotnet/cli/commit/357fd3d) Fixing restore so that it respects the verbosity param. The problem was that ConsoleLoggerParameters was overwritting whatever was coming through the command line.
+* [`[357fd3d]`](https://github.com/dotnet/cli/commit/357fd3d) Fixing restore so that it respects the verbosity param. The problem was that ConsoleLoggerParameters was overwriting whatever was coming through the command line.
```

Comment:
```suggestion
* [`[357fd3d]`](https://github.com/dotnet/cli/commit/357fd3d) Fixing restore so that it respects the verbosity param. The problem was that ConsoleLoggerParameters was overwritting whatever was coming through the command line.
```